src/txmempool.cpp

```diff
@@ -216,6 +240,17 @@ bool CTxMemPool::CalculateMemPoolAncestors(const CTxMemPoolEntry &entry, setEntr
     return true;
 }
 
+bool CTxMemPool::CalculateMemPoolAncestors(const CTxMemPoolEntry &entry, setEntries &setAncestors,
+                                           uint64_t limitAncestorCount, uint64_t limitAncestorSize,
+                                           uint64_t limitDescendantCount, uint64_t limitDescendantSize,
+                                           std::string &errString, bool fSearchForParents /* = true */) const
+{
+    std::vector<CTxMemPoolEntry::CTxMemPoolEntryRef> entry_vec{std::cref(entry)};
```

Comment:
if you update to a span, you get rid of an allocation here!

Reply:
GONE! 👐